Sources/Workspace/Workspace.swift

```diff
@@ -3402,6 +3402,24 @@ extension Workspace {
          // remove the local copy
          try registryDownloadsManager.remove(package: dependency.packageRef.identity)
      }
+
+     /// Accept an identity change that we raise a `RegistryError` for.
+     public func acceptIdentityChange(
+         package: PackageIdentity.RegistryIdentity,
+         version: Version,
+         signingEntity: SigningEntity,
+         observabilityScope: ObservabilityScope,
```

Comment:
I don't know if we want to add `registry: Registry` to the argument list--this is determined by whether we think `PackageSigningEntityStorage` should keep track of where `SigningEntity` comes from (i.e., who tells about the `SigningEntity`?). Right now it doesn't, but I am thinking maybe it should. It's good info to have I think.

Reply:
yes that seems to be good so one registry cannot step on another one

Reply:
Currently, I don't think the client knows the registry necessarily, so adding that may make the API unusable.